one_fm/public/js/desk.js

```diff
@@ -42,43 +42,8 @@ let improve_my_erp = () => {
 	let improveBTN = document.createElement('a');
 	improveBTN.classList = "btn btn-default btn-xs improve-my-erp";
 	improveBTN.textContent = "Improve";
+  improveBTN.href = '/helpdesk/tickets/new'
 	document.querySelector(".form-inline.fill-width.justify-content-end").prepend(improveBTN);
-	$('.improve-my-erp').on('click', function() {
```

Comment:
You removed a block of code here, was that intentional ?

Reply:
Yes, we just needed to redirect instead of showing a dialog. So the dialog code was useless that's why I removed that block of code